integration/integration_test.go

```diff
@@ -19,23 +19,23 @@ func TestIntegration(t *testing.T) {
 
 var _ = Describe("Happy path", func() {
 	BeforeSuite(func() {
-		buildCmd := exec.Command("docker-compose", "build")
+		buildCmd := exec.Command("docker compose", "build")
 		session, err := gexec.Start(buildCmd, GinkgoWriter, GinkgoWriter)
 		Expect(err).ShouldNot(HaveOccurred())
 		Eventually(session, 60).Should(gexec.Exit(0))
 	})
 
 	BeforeEach(func() {
 		upCmd := exec.Command(
-			"docker-compose", "up", "--detach", "--force-recreate",
+			"docker compose", "up", "--detach", "--force-recreate",
```

Comment:
```suggestion
			"docker", "compose", "up", "--detach", "--force-recreate",
```

Reply:
This change should work, with it in a single string it's looking for an executable with a space in the file name

Reply:
Ah fabulous - I knew it would be something tiny like this! Thanks @jfharden, I'll give it a whirl

Reply:
aaaaand a green build 🥳